astropy/io/fits/diff.py

```diff
@@ -262,6 +276,9 @@ def __init__(self, a, b, ignore_keywords=[], ignore_comments=[],
         ignore_blank_cards : bool, optional
             Ignore all cards that are blank, i.e. they only contain
             whitespace (default: True).
+
+        tolerance : float, optional
```

Comment:
But `atol` is added in between existing parameters so it will permanently change the function signature. Say you call the function with positional parameters only then the old `ignore_blanks` will now be interpreted as `atol`.

But you're right for the `tolerance` -> `rtol` change.